config/packages/doctrine.yaml

```diff
@@ -0,0 +1,22 @@
+doctrine:
+    dbal:
+        # configure these for your database server
+        driver: 'pdo_mysql'
+        server_version: '5.7'
+        charset: utf8mb4
+        default_table_options:
+            charset: utf8mb4
+            collate: utf8mb4_unicode_ci
+
+        url: '%env(resolve:DATABASE_URL)%'
```

Comment:
When/if we start using a database for storing data sets and transformed data, we probably want to set `schema_filter` here to exclude custom data source/transform tables (cf. https://symfony.com/doc/master/bundles/DoctrineMigrationsBundle/index.html#manual-tables). Alternatively, we could use another database connection for the actual data wrangling.